tests/settings/SectionFillingTest.py

```diff
@@ -87,6 +89,18 @@ def test_fill_section(self):
         self.assertEqual(new_section['key'].value, 'val')
         self.assertEqual(len(new_section.contents), 3)
 
```

Comment:
please add a comment here that you are testing the deprecation of calling `acquire_settings`